core/block.ts

```diff
@@ -570,6 +570,7 @@ export class Block implements IASTNodeLocation, IDeletable {
    * @return The block (if any) that surrounds the current block.
    */
   getSurroundParent(): this|null {
+    /* eslint-disable-next-line @typescript-eslint/no-this-alias */
```

Comment:
it would be sweet if this rule were smart enough to only trigger if the alias was used inside a function. ah well.